### DISCUSSION TITLE
Docker Prod Deployment

### DISCUSSION THREAD

**Discussion by msbrown47** [tethysplatform/tethys#844] [General]
Hey,

Our team has been working on creating a deployable, production-ready version of Tethys. We have faced a few challenges in doing so. We have gone through the Docker tutorial, and we also encountered issues using the code hosted on GitHub and the pre-built Docker image. Is there a way we can tag up to walk through these issues?

Thanks,

**Comment by swainn** [comment]
Sure thing @msbrown47. I'll reach out to you via email.
